README.md

```diff
@@ -130,24 +137,27 @@ cd Unvanquished/
 Produced files will be stored in a new directory named `build`.
 
 ```sh
-cmake -H. -Bbuild
-cmake --build build -- -j4
+mkdir build; cd build
+cmake ..
+make -j4
 ```
 
 ### MSYS2
 
 ```sh
-cmake -H. -Bbuild -G "MSYS Makefiles"
-cmake --build build -- -j4
+mkdir build; cd build
+cmake .. -G "MSYS Makefiles"
+make -j4
```

Comment:
I guess this is the most “old cmake-compatible” way, otherwise less old cmake can do:

```sh
cmake -S. -Bbuild -G"MSYS Makefiles"
cmake --build build --parallel 4
```

Which is totally generator agnostic.

If one wants to run build commands in folder he can do:

```sh
cmake -S. -Bbuild -G"MSYS Makefiles"
cmake --build build --parallel 4
```

or:

```sh
mkdir -p build ; cd build
cmake -S.. -B. -G"MSYS Makefiles"
cmake --build . --parallel 4
```

or:

```sh
mkdir -p build ; cd build
cmake .. -G"MSYS Makefiles"
cmake --build . --parallel 4
```